src/dialer/utils.js

```diff
@@ -0,0 +1,45 @@
+'use strict'
+
+const isPrivate = require('libp2p-utils/src/multiaddr/is-private')
+
+// TODO: Move to libp2p-utils?
```

Comment:
Yeah, I think this would be useful for other modules that would avoid needing to use libp2p as a dependency.